app/scripts/modules/netflix/availability/availability.less

```diff
@@ -149,6 +152,9 @@
       padding: 15px 20px;
       color: #fff;
     }
+    .glyphicon {
```

Comment:
I missed this the first time around.  We should be moving to font-awesome.  Apologies.